packages/terra-menu/src/MenuItem.module.scss

```diff
@@ -7,9 +7,29 @@
     &:not(.is-disabled) {
       cursor: pointer;
 
-      &:hover,
+      &:hover {
+        background-color: var(--terra-menu-item-hover-background-color, #ebf6fd);
+      }
+
+      &:hover .checkmark {
+        color: var(--terra-menu-item-checkmark-hover-icon-color);
+      }
+
+      &:hover .chevron {
+        color: var(--terra-menu-item-chevron-hover-icon-color);
+      }
+
       &:focus {
-        background-color: var(--terra-menu-item-hover-color, #ebf6fd);
+        background-color: var(--terra-menu-item-focus-background-color, #ebf6fd);
+        outline: var(--terra-menu-item-focus-outline, -webkit-focus-ring-color auto 5px); //browser default fallback. if no fallback is provided, outline does not render.
```

Comment:
Would it make sense to leave the fallback empty? 
e.g. ` outline: var(--terra-menu-item-focus-outline);`

Reply:
The outline does not render if the fallback is empty. I'm not sure why - I'm open to better alternatives